webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue

```diff
@@ -183,20 +183,22 @@
       </a>
     </div>
 
-    <v-card-text class="peopleCardBody align-center py-1">
-      <a
-        :href="url"
-        :title="user.fullname"
-        :class="usernameClass"
-        class="userFullname font-weight-bold">
-        {{ user.fullname }}
-        <span v-if="user.external == 'true' " class="externalFlagClass">
-          {{ $t('peopleList.label.external') }}
-        </span>
-      </a>
+    <v-card-text class="peopleCardBody align-center py-0 py-sm-1">
+      <v-sheet :height="isMobile &&'47px' || '26px'" class="mt-0">
```

Comment:
Add **height** to computed props, please!

Reply:
done